src/t_zset.c

```diff
@@ -2257,26 +2257,14 @@ int zuiFind(zsetopsrc *op, zsetopval *val, double *score) {
         return 0;
 
     if (op->type == OBJ_SET) {
-        if (op->encoding == OBJ_ENCODING_INTSET) {
-            if (zuiLongLongFromValue(val) &&
-                intsetFind(op->subject->ptr,val->ell))
-            {
-                *score = 1.0;
-                return 1;
-            } else {
-                return 0;
-            }
-        } else if (op->encoding == OBJ_ENCODING_HT) {
-            dict *ht = op->subject->ptr;
-            zuiSdsFromValue(val);
-            if (dictFind(ht,val->ele) != NULL) {
-                *score = 1.0;
-                return 1;
-            } else {
-                return 0;
-            }
+        zuiSdsFromValue(val);
+        /* There is no reason for zuiFind to go into the internals of the SET.
+         * It can simply use setTypeIsMember and don't care about encoding. */
```

Comment:
This is not a relevant code comment IMO. Perhaps it can be relevant as a commit message.

```suggestion
```